### PR TITLE
Vwan availability zones

### PR DESCRIPTION
**Pull request by jhajduk-microsoft**
## Overview/Summary

Adding availability zones to the Azure Firewall that is part of the managed VWAN hub template
## This PR fixes/adds/changes/removes

1. Implementing functionality consistent with supported features of Azure Firewall used with Azure Managed virtual wan

### Breaking Changes

None

## Testing Evidence

[https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fjhajduk-microsoft%2FEnterprise-Scale%2Fvwan-availability-zones%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Fjhajduk-microsoft%2FEnterprise-Scale%2Fvwan-availability-zones%2FeslzArm%2Feslz-portal.json](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fjhajduk-microsoft%2FEnterprise-Scale%2Fvwan-availability-zones%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Fjhajduk-microsoft%2FEnterprise-Scale%2Fvwan-availability-zones%2FeslzArm%2Feslz-portal.json)

See: https://github.com/Azure/Enterprise-Scale/pull/960#issuecomment-1121301064

## As part of this Pull Request I have

- [Y] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [ ] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [Y] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [Y] Performed testing and provided evidence.
- [Y] Updated relevant and associated documentation.
- [ ] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

**Comment by jhajduk-microsoft** [comment]
@jtracey93 - Let me know if other changes are needed as part of this

**Comment by jhajduk-microsoft** [comment]
@jtracey93 - I believe I have added in the correct places now, but when I go to the firewall and look at properties, the availability zones property is empty: https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fjhajduk-microsoft%2FEnterprise-Scale%2Fvwan-availability-zones%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Fjhajduk-microsoft%2FEnterprise-Scale%2Fvwan-availability-zones%2FeslzArm%2Feslz-portal.json

File: docs/wiki/Whats-new.md [hunk 1]
```diff
@@ -37,6 +38,22 @@ This article will be updated as and when changes are made to the above and anyth
...
+### May 2022
+
+#### Docs
+
+#### Tooling
+
+- Updating Azure Firewall to use Availability Zones inside of Azure Virtual WAN Secure Hub: [Azure Firewall Updates](https://azure.microsoft.com/blog/boost-your-network-security-with-new-updates-to-azure-firewall/) and [Azure Firewall with Availability Zones](https://docs.microsoft.com/azure/virtual-wan/virtual-wan-faq#how-are-availability-zones-and-resiliency-handled-in-virtual-wan)
+
+### Policy
+
```
Comment (jtracey93) [inline docs/wiki/Whats-new.md:50]:
```suggestion

- *No updates, yet.*

```

File: docs/wiki/Whats-new.md [hunk 2]
```diff
@@ -37,6 +38,22 @@ This article will be updated as and when changes are made to the above and anyth
 
 Here's what's changed in Enterprise Scale:
 
+### May 2022
+
+#### Docs
+
```
Comment (jtracey93) [inline docs/wiki/Whats-new.md:44]:
```suggestion

- *No updates, yet.*

```

**Review by jtracey93 — CHANGES_REQUESTED** [review]
A few small additions

File: eslzArm/eslz-portal.json [hunk 3]
```diff
@@ -1553,7 +1553,7 @@
                             "multiselect": true,
                             "selectAll": true,
                             "filter": true,
-                            "visible": "[if(equals(steps('esConnectivityGoalState').esHub, 'vhub'), and(equals(steps('esConnectivityGoalState').esAzFw,'Yes'), contains(split('canadacentral,centralus,eastus,eastus2,southcentralus,westus2,francecentral,germanywestcentral,northeurope,westeurope,uksouth,southafricanorth,japaneast,southeastasia,australiaeast', ','), steps('esConnectivityGoalState').esNwLocation)), false)]",
+                            "visible": "[if(equals(steps('esConnectivityGoalState').esHub, 'vhub'), or(equals(steps('esConnectivityGoalState').esHub, 'vwan'), and(equals(steps('esConnectivityGoalState').esAzFw,'Yes'), contains(split('canadacentral,centralus,eastus,eastus2,southcentralus,westus2,francecentral,germanywestcentral,northeurope,westeurope,uksouth,southafricanorth,japaneast,southeastasia,australiaeast', ','), steps('esConnectivityGoalState').esNwLocation)), false)]",
```
Comment (jtracey93) [inline eslzArm/eslz-portal.json:1556]:
```suggestion
                            "visible": "[if(equals(steps('esConnectivityGoalState').esHub, 'vhub'), or(equals(steps('esConnectivityGoalState').esHub, 'vwan')), and(equals(steps('esConnectivityGoalState').esAzFw,'Yes'), contains(split('canadacentral,centralus,eastus,eastus2,southcentralus,westus2,francecentral,germanywestcentral,northeurope,westeurope,uksouth,southafricanorth,japaneast,southeastasia,australiaeast', ','), steps('esConnectivityGoalState').esNwLocation)), false)]",
```

File: eslzArm/fairfaxeslz-portal.json [hunk 4]
```diff
@@ -1435,7 +1435,7 @@
                             "multiselect": true,
                             "selectAll": true,
                             "filter": true,
-                            "visible": "[if(equals(steps('esConnectivityGoalState').esHub, 'vhub'), and(equals(steps('esConnectivityGoalState').esAzFw,'Yes'), contains(split('canadacentral,centralus,eastus,eastus2,southcentralus,westus2,francecentral,germanywestcentral,northeurope,westeurope,uksouth,southafricanorth,japaneast,southeastasia,australiaeast', ','), steps('esConnectivityGoalState').esNwLocation)), false)]",
+                            "visible": "[if(equals(steps('esConnectivityGoalState').esHub, 'vhub'), or(equals(steps('esConnectivityGoalState').esHub, 'vwan'), and(equals(steps('esConnectivityGoalState').esAzFw,'Yes'), contains(split('canadacentral,centralus,eastus,eastus2,southcentralus,westus2,francecentral,germanywestcentral,northeurope,westeurope,uksouth,southafricanorth,japaneast,southeastasia,australiaeast', ','), steps('esConnectivityGoalState').esNwLocation)), false)]",
```
Comment (jtracey93) [inline eslzArm/fairfaxeslz-portal.json:1438]:
```suggestion
                            "visible": "[if(equals(steps('esConnectivityGoalState').esHub, 'vhub'), or(equals(steps('esConnectivityGoalState').esHub, 'vwan')), and(equals(steps('esConnectivityGoalState').esAzFw,'Yes'), contains(split('canadacentral,centralus,eastus,eastus2,southcentralus,westus2,francecentral,germanywestcentral,northeurope,westeurope,uksouth,southafricanorth,japaneast,southeastasia,australiaeast', ','), steps('esConnectivityGoalState').esNwLocation)), false)]",
```

**Comment by jtracey93** [comment]
Testing Evidence:
image
image
image
image

**Pull request merged**